packages/optimizer/lib/transformers/SeparateKeyframes.js

```diff
@@ -36,6 +43,11 @@ const allowedKeyframeProps = new Set([
 class SeparateKeyframes {
   constructor(config) {
     this.log_ = config.log.tag('SeparateKeyframes');
+    if (config.compress === false || config.compress === 'false') {
```

Comment:
Why can this be a string? Assuming that's by design, you could probably simplify with `if (String(config.compress) === 'false')`.

Could also shorten further to
```js
const compress = String(config.compress) !== 'false';
this.stringifyOptions_ = compress ? OPTIONS_COMPRESS : OPTIONS_PRETTY_PRINT;
```
and get rid of the `if` all together.

Reply:
This was to support strings passed via the CLI, but that's the wrong place to handle it. Changed to simple boolean.